### ISSUE TITLE
video instead of still

### ISSUE THREAD

**Issue by Neon22** [mpetroff/pannellum#10]
Is it possible for this superb implementation to use a video instead of a stil image ?
The video would be pretransformed such as:
http://sphericam.com/portfolio/octocopter-in-la-720p-equirect-crop/

**Comment by mpetroff** [comment]
Partial support in 2.1.0

**Comment by pc9geek** [comment]
When you say partial support, what do you mean? As yet I have been unable to get a webm video to render, it displays black while the audio plays.

**Comment by mpetroff** [comment]
By partial support, I mean it should play the video, but there are no controls for play, pause, seeking, etc. The video support it poorly tested; what you're describing sounds like a bug.

**Comment by pc9geek** [comment]
Thanks I'll see if I can work it out

**Comment by mpetroff** [comment]
I just implemented proper video support in eb5a0891cdac19a2bafa72349d4decd0cbfd8e3b (it breaks any existing videos, though). It will be in the next release. Instead of reinventing the wheel, I wrote a [Video.js](http://videojs.com/) [plugin](https://github.com/mpetroff/pannellum/blob/master/utils/video/videojs-pannellum-plugin.js) that uses Pannellum. Until I add a proper example, the following will have to suffice:

```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>Pannellum Video Demo</title>
  <link href="https://vjs.zencdn.net/5.0/video-js.css" rel="stylesheet" type="text/css">
  <link href="pannellum.css" rel="stylesheet" type="text/css">
</head>
<body>

<video id="example-video" class="video-js vjs-default-skin vjs-big-play-centered"
    controls preload="none" width="640" height="320" poster="poster.jpg" data-setup="{}">
  <source src="video.webm" type="video/webm"/>
  <source src="video.mp4" type="video/mp4"/>
  <p class="vjs-no-js">To view this video please enable JavaScript.</p>
</video>

<script src="https://vjs.zencdn.net/5.0/video.js"></script>
<script src="pannellum.js"></script>
<script src="videojs-pannellum-plugin.js"></script>

<script>
videojs('example-video', {
    plugins: {
        pannellum: {}
    }
});
</script>

</body>
</html>
```

**Comment by pc9geek** [comment]
Great thanks, I'll have a look. I appreciate your effort and the quality of your work
